DNN Platform/Library/Entities/Users/IUserController.cs

```diff
@@ -107,5 +107,12 @@ public interface IUserController
         /// in background threads or scheduler jobs. 
         /// </remarks>
         string GetUserProfilePictureUrl(int portalId, int userId, int width, int height);
+
+        /// <summary>
+        /// Check username against
```

Comment:
Nit: Please complete the comment ✍️

Reply:
Thnx, too fast too furious, how could I missed that 😮